src/api/app/models/staging/excluded_request_not_found.rb

```diff
@@ -0,0 +1,9 @@
+class Staging::ExcludedRequestNotFound < APIError
+  attr_reader :message
```

Comment:
you really don't need all that. APIError already brings that. Including the render_error if caught in application_controller. All you need to have is `setup 'invalid_request'` in the class.

Reply:
@coolo is that the change request or just a comment? Please help @danidoni to get this PR done.

Reply:
Overriding initialize of APIError to is basically just making a StandardError again - and I said previously this is wrong.

Reply:
Done